packages/api-client/src/views/Request/RequestSection/RequestAuth.vue

```diff
@@ -276,7 +282,7 @@ const unselectAuth = (id: string) => {
               <RequestAuthDataTableInput
                 :id="`http-bearer-token-${scheme.uid}`"
                 :modelValue="scheme.value"
-                placeholder="QUxMIFlPVVIgQkFTRSBBUkUgQkVMT05HIFRPIFVT"
+                placeholder="Token"
```

Comment:
i felt that the previous placeholder was too close from a real value making it hard to distinguish, any thougths? @amritk

Reply:
it was an easter egg, but i think we can move this easter egg to something else :)